R/auditCommonFunctionsBayesian.R

```diff
@@ -348,6 +348,63 @@
   }
 }
 
+.jfaPlotSequentialAnalysis <- function(options, jaspResults, parentState, parentContainer, positionInContainer, sample, evaluationOptions) {
```

Comment:
I prefer to have an indication that it is about sampling here, so maybe `.jfaPlotSequentialAnalysisEvaluation()` or `.jfaPlotSequentialAnalysisSampling()`.